src/BaGet/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -98,6 +101,15 @@ public static IServiceCollection AddBaGetContext(this IServiceCollection service
                 options.UseSqlServer(databaseOptions.Value.ConnectionString);
             });
 
+            services.AddDbContext<PostgresSqlContext>((provider, options) =>
+            {
+                var databaseOptions = provider.GetRequiredService<IOptions<BaGetOptions>>()
```

Comment:
I recently tweaked the configuration model. This line should now be: `provider.GetRequiredService<IOptions<DatabaseOptions>>();`

Reply:
I changed it to `IOptions` but the two providers above use `IOptionsSnapshot`, which one is correct?

Reply:
You should use `IOptionsSnapshot` so that the database's configs can be reloaded automatically